phpunit.xml.dist

```diff
@@ -1,33 +1,32 @@
-<!-- 
+<!--
 	PHPUnit configuration for SilverStripe
-
-	Requires PHPUnit 3.5+
-
-	Usage: 
+	Requires PHPUnit 5+
+	Usage:
 	 - "phpunit": Runs all tests in all folders
 	 - "phpunit framework/tests/": Run all tests of the framework module
 	 - "phpunit framework/tests/filesystem": Run all filesystem tests within the framework module
 	 - "phpunit framework/tests/filesystem/FolderTest.php": Run a single test
 	 - "phpunit <dash><dash>coverage-html assets/": Generate coverage report (replace <dash> with "-", requires xdebug)
```

Comment:
I think we should update the usage examples now:

```diff
- - "phpunit framework/tests/": Run all tests of the framework module
+ - "vendor/bin/phpunit vendor/silverstripe/framework/tests/": Run all tests of the framework module
```

This equally could be said for silverstripe/installer as well

Reply:
I actually updated the comment block by copying it from silverstripe/installer, ha! Should have read it too; shame.